site/content/docs/table-of-contents.md

```diff
@@ -31,3 +31,34 @@ The table of contents will only include headings placed below the `Table of cont
 ## Another heading
 
 ### Yet another heading
+
+***
+
+## Right Hand Side (RHS)
+
+Flowershow supports adding a Table of Contents (ToC) sidebar (RHS) to your pages which can be easily setup in your config file.
+
+### ToC on all pages
+To enable ToC on all the markdown pages (except homepage), simply set the table of contents variable to true in your config file as seen below:
+
+```js
+// config.js
+
+const config = {
+  tableOfContents: true
+}
+```
+
+### ToC for custom pages/folders
+
+Sometimes there is a need for table of contents only on specific pages, for example docs or blog pages. Flowershow also supports enabling ToC only for these specific defined page/pages by passing that directory or path from your content folder in an array as seen below.
+
+```js
+// config.js
+
+const config = {
+  tableOfContents: ["docs", "blog", "about"]
+}
```

Comment:
Let's keep this always of the same type. It's rather strange that it can be either boolean or list of strings. Also, up until know we used config to set default values that could later be changed in frontmatter for specific pages. Let's maybe stick with this approach?

Reply:
So does the following look correct maybe ?

```js
const config = {
  tableOfContents: ["all"]
}

Reply:
@olayway I may not have understood here. Do we want to be able to have something like below in frontmatter for specific pages ?
```md
---
tableOfContents: true
---
```

Reply:
I'd do it the same way we handle e.g. showing edit this page link, wich is enabling/disabling by default using boolean value in config, with the possibility to override this value in frontmatter.